skills_ml/datasets/README.md

```diff
@@ -0,0 +1,20 @@
+datasets
+-----------
```

Comment:
The one sentence introduction to the datasets module would be good here as well, before going into the county example.